code/modules/ordeals/_ordeal.dm

```diff
@@ -33,14 +35,16 @@
 /datum/ordeal/proc/Run()
 	start_time = ROUNDTIME
 	SSlobotomy_corp.current_ordeals += src
-	priority_announce(annonce_text, name, sound='sound/effects/meltdownAlert.ogg')
+	priority_announce(announce_text, name, sound='sound/effects/meltdownAlert.ogg')
 	/// If dawn started - clear suppression options
 	if(level == 1 && !istype(SSlobotomy_corp.core_suppression))
 		SSlobotomy_corp.ResetPotentialSuppressions()
-	if(annonce_sound)
-		for(var/mob/M in GLOB.player_list)
-			if(M.client)
-				M.playsound_local(get_turf(M), annonce_sound, 35, 0)
+	for(var/mob/M in GLOB.player_list)
```

Comment:
can you please change this into NOT a single-letter variable?

Reply:
This isn't really a big deal but I certainly can.